apps/gitness/src/pages/project-settings/components/FormDialogProjectDelete.tsx

```diff
@@ -27,9 +27,15 @@ interface FormProjDeleteProps {
   handleDeleteProject: () => void
   isDeleteSuccess: boolean
   isDeleting: boolean
+  deleteError: string | null
 }
 //delete project form with dialog
-export const FormDialogProjectDelete = ({ handleDeleteProject, isDeleteSuccess, isDeleting }: FormProjDeleteProps) => {
+export const FormDialogProjectDelete = ({
```

Comment:
naming should follow normal conventions?

Reply:
fixed